lebedigital/calibration/calibrationWorkflow.py

```diff
@@ -0,0 +1,202 @@
+                          info="Slope of the graph",
+                          prior=('normal', {'mean': loc_E,
+                                            'std': scale_E}))
+
+    problem.add_parameter('sigma', 'likelihood',
+                          tex=r"$\sigma$",
+                          info="Std. dev, of 0-mean noise model",
+                          prior=('uniform', {'low': low_sigma,
+                                             'high': high_sigma}))
+
+    # Load the forward model and add it to the Inverse problem
+    linear_elasticity = LinearElasticityCylinder("linear_elasticity_cylinder")
+    problem.add_forward_model(linear_elasticity)
+
+    # ============================================
+    #     Add test data to the Inverse Problem
+    # ============================================
+
+    y_test = exp_output['stress'] * (np.pi * (exp_output['diameter'] / 2) ** 2)  # in kN
+    # add the experimental data
+    problem.add_experiment(
+        experiment_name,
+        fwd_model_name="linear_elasticity_cylinder",
+        sensor_values={
+            'nu': 0.2,
+            'height': exp_output['height'],
+            'radius': exp_output['diameter'] / 2,
+            'displacement_list': exp_output['displacement'],
+            linear_elasticity.output_sensor.name: y_test,
+        },
+    )
+
+    # ==============================================
+    #       Add likelihood model(s)
+    # ==============================================
+
+    # add the likelihood model to the problem
+    problem.add_likelihood_model(
+        GaussianLikelihoodModel(
+            prms_def="sigma",
+            experiment_name=experiment_name,
+            model_error="additive",
+            name="SimpleLikelihoodModel",
+        )
+    )
+
+    # give problem overview
+    problem.info()
+
+    # ==============================================
+    #        Export knowledge graph
+    # ==============================================
+
+    # create the knowledge graph and print it to file
+    #dir_path = os.path.dirname(__file__)
+    dir_path = calibrated_data_path
+    basename_owl = os.path.basename(__file__).split(".")[0] + ".owl"
+    knowledge_graph_file = os.path.join(dir_path, basename_owl)
+    export_knowledge_graph(problem, knowledge_graph_file, data_dir=dir_path)
+
+    # ========================================================================
+    #        Solve problem with inference engine and write results to graph
+    # ========================================================================
+
+    # run inference step using emcee
+    emcee_solver = EmceeSolver(problem, show_progress=True)
+
+    if mode == 'cheap':
+        inference_data = emcee_solver.run_mcmc(
+            n_walkers=4,
+            n_steps=1,
+            n_initial_steps=4,
+        )
+    else:
+        inference_data = emcee_solver.run_mcmc(
+            n_walkers=5,
+            n_steps=40,
+            n_initial_steps=5,
```

Comment:
Does this setup result in an accurate estimate?

Reply:
That would mean we should (in addition to the accuracy) add a seed for the random number generator.

Reply:
Yes the walker and no steps are not enough. Need more. Also seed is needed, atleast for the tests. I will see how I can do so.